src/style-spec/reference/v8.json

```diff
@@ -321,6 +331,11 @@
       "default": "mapbox",
       "doc": "The encoding used by this source. Mapbox Terrain RGB is used by default"
     },
+    "volatile": {
+      "type": "boolean",
+      "default": false,
+      "doc": "Setting this flag to true indicates that the source tiles shall not be stored in the local cache." 
```

Comment:
@pozdnyakov we should update the documentation here too so that the language is the same everywhere